pkg/controller/node/node_controller.go

```diff
@@ -955,18 +1092,19 @@ func (ctrl *Controller) setDesiredMachineConfigAnnotation(nodeName, currentConfi
 // getAllCandidateMachines returns all possible nodes which can be updated to the target config, along with a maximum
 // capacity.  It is the reponsibility of the caller to choose a subset of the nodes given the capacity.
 func getAllCandidateMachines(pool *mcfgv1.MachineConfigPool, nodesInPool []*corev1.Node, maxUnavailable int) ([]*corev1.Node, uint) {
-	targetConfig := pool.Spec.Configuration.Name
-	unavail := getUnavailableMachines(nodesInPool)
-	// If we're at capacity, there's nothing to do.
+	unavail := getUnavailableMachines(nodesInPool, pool)
+	klog.Infof("Unavailable node count: %d, maxUnavailable: %d", len(unavail), maxUnavailable)
```

Comment:
nit: similar to above, slightly noisy, considering dropping verbosity of (mostly) debug useful logs